packages/react-components/src/link/src/Link.css

```diff
@@ -118,13 +118,13 @@
 /* TEXT | STATE | DISABLED | HOVER */
 .o-ui-link[disabled]:hover .o-ui-link-text,
 .o-ui-link[disabled].o-ui-link-hover .o-ui-link-text {
-    text-decoration: none;
+    text-decoration: underline;
 }
 
 /* TEXT | STATE | DISABLED | FOCUS */
 .o-ui-link[disabled]:focus .o-ui-link-text,
 .o-ui-link[disabled].o-ui-link-focus .o-ui-link-text {
-    text-decoration: none;
+    text-decoration: underline;
```

Comment:
I think it would be better if you don't want assume in CSS what is the default underline style.

I would prefer if it's all controlled by our default value in the TextLink component. 

Could we do like we do for variants and render a css class for each underline values?

Something like:

- solid -> solid
- dotted -> dotted
- none -> no-underline

Names are only a suggestion you might have a better proposition